src/Propel/Common/Config/XmlToArrayConverter.php

```diff
@@ -36,17 +36,17 @@ public static function convert($xmlToParse)
         $isFile = file_exists($xmlToParse);
 
         //Empty xml file returns empty array
```

Comment:
The comment does not make sense here. Would you mind removing it?

Reply:
moved to proper position